Source.cpp

```diff
@@ -0,0 +1,43 @@
+#include <Header.h>
+
+int main()
+{
+	Party A;
+	A.Setter("November 27 ", "Birthday party");
+	A.InputParty();
+	cout << "Guests:" << endl << endl;
+	const int length = 7;
+	vector<Guest> guest{
+		Guest(346, "Michael", 18, "Lviv", 8776879),
+		Guest(128, "George", 21, "Chernigiv", 5647689),
+		Guest(48, "Sam", 17, "Lviv", 7658908),
+		Guest(568, "Amanda", 20, "Ternopil", 1287677),
+		Guest(167, "Sara", 19, "Sambir", 8805535),
+		Guest(13, "Edvard", 20, "Kalush", 7897876),
+		Guest(777, "Steav", 22, "Kyiv", 3409056)
+	};
```

Comment:
is pushback a joke for you?

Reply:
Pushback is not appropriate for this situation, beacause i need to create a several guests and fill their information. Since I will still have to fill information manually, pushback is not needed here at all. Only if i want to add a new object to the vector.